addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CrossDomainMisconfigurationUnitTest.java

```diff
@@ -81,7 +80,8 @@ public void shouldRaiseAlertIfCorsAllowOriginHeaderIsTooPermissive() {
         assertThat(
                 alertsRaised.get(0).getEvidence(),
                 is(HttpHeader.ACCESS_CONTROL_ALLOW_ORIGIN + ": *"));
-        assertThat(alertsRaised.get(0).getOtherInfo(), containsString("CORS misconfiguration"));
+        //        assertThat(alertsRaised.get(0).getOtherInfo(), containsString("CORS
```

Comment:
Why?

Reply:
Because I was troubleshooting and forgot to restore it. Good catch :+1: